### ISSUE TITLE
Does it support arm or x86 architecure ?

### ISSUE THREAD

**Issue by colorlight** [rk700/uniFuzzer#1]
When I test it with the x86 version binary, the dynamic lib symbol's address isn't updated in the got table. Have you implemented the arm and x86 functions?

**Comment by rk700** [comment]
Some symbol types on x86 are not supported yet, such as IFUNC which needs runtime resolution. ARM should be fine I think, though I haven't tested.

**Comment by colorlight** [comment]
Yes, I found that the symbol type is IFUNC. By the way, will you implement the IFUNC
